sdk/pinocchio/src/entrypoint/lazy.rs

```diff
@@ -136,11 +133,12 @@ impl InstructionContext {
     #[inline(always)]
     pub unsafe fn new_unchecked(input: *mut u8) -> Self {
         Self {
-            input,
             // SAFETY: The first 8 bytes of the input buffer represent the
             // number of accounts when serialized by the SVM loader.
+            input: unsafe { input.add(core::mem::size_of::<u64>()) },
```

Comment:
The doc comment for `input` needs to be updated now, I think.

Would it also make sense to rename `input` to, maybe, `unprocessed_input` to make it more obvious that it is not the original data passed into `new()`?